Wikimate.php

```diff
@@ -741,7 +741,9 @@ public function getError()
  * @since   1.0.0  August 2021
  * @link    https://www.php.net/manual/en/class.runtimeexception.php
  */
-class WikimateException extends RuntimeException {}
+class WikimateException extends RuntimeException
+{
+}
```

Comment:
Ugh. I wish the recommendations included an exception (heh) for the empty body case.